pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -296,9 +339,9 @@ def make_package(args):
         with open(args.intent_filters) as fd:
             args.intent_filters = fd.read()
 
-    args.add_activity = args.add_activity or []
-
-    args.activity_launch_mode = args.activity_launch_mode or ''
+    if get_bootstrap_name() == "sdl2":
+        args.add_activity = args.add_activity or []
+        args.activity_launch_mode = args.activity_launch_mode or ''
```

Comment:
kinda out of scope, but:
people tend to misused Python ternary operator. It's fine here because you just move the lines, but I find it pretty misleading and can lead to unsuspected bugs

Reply:
Yeah I touched as little as possible to not make things even more complex :grimacing: but keep those comments coming, so I can look into them once I refactor `build.py` in a follow-up request!